featureflags.go

```diff
@@ -243,6 +219,11 @@ func (poller *FeatureFlagsPoller) GetFeatureFlags() []FeatureFlag {
 func (poller *FeatureFlagsPoller) request(method string, endpoint string, requestData []byte, headers [][2]string) (*http.Response, error) {
 
 	url := poller.Endpoint + "/" + endpoint + "/?token=" + poller.projectApiKey + ""
+
+	if endpoint == "decide" {
```

Comment:
@neilkakkar @rcmarron do we need the `token` param for the decide v2 endpoint? I've seen it without that param in a server library but I'm not 100% sure

Reply:
Good question, you can send it in the POST body too! With the key `api_key`